### DISCUSSION TITLE
Deta Cloud已经迁移至Deta Space，Deta Cloud已经无法新建项目

### DISCUSSION THREAD

**Discussion by MrL646** [walinejs/waline#1822] [Q&A] [answered]
官方迁移公告：https://deta.space/from-cloud
Deta官方将Deta Cloud迁移到了Deta Space，原来的Deta Cloud已经无法新建项目
原有的Deta一键部署失效 https://github.com/walinejs/deta-starter
现有的Deta Cloud无法使用原来的配置部署，Deta Base正常工作

**Comment by lizheming** [comment, marked as answer]
https://deta.space/discovery/@lizheming/waline 已经新增了 Waline App 了，可以直接在 Deta Discovery 里面一键安装
